### PR TITLE
docs: add testing strategy 

### PR DESCRIPTION
**Pull request by DlgSHi**
Add test strategy

### CODE REVIEW

File: cypress/README.md [hunk 1]
```diff
@@ -1,4 +1,30 @@
-# Cypress Cucumber tests for Carbon storybook
+# Testing Strategy
+
+## Unit Testing
+Unit tests are writing / maintenancing by the developers
+
+## Manual Testing
+### Components:
+1. Every component should be fully tested using functional testing / UI testing / Internationalisation testing
+2. Should be tested all stories for dedicated component (default / classic / multiple / validation / themes etc.), notes section and accessibility section.
```
Comment (KatarzynaQA) [inline cypress/README.md]: Please, remove the blank space, it'll look better :)

File: cypress/README.md [hunk 2]
```diff
@@ -1,4 +1,30 @@
-# Cypress Cucumber tests for Carbon storybook
+# Testing Strategy
+
+## Unit Testing
+Unit tests are writing / maintenancing by the developers
```
Comment (jamime) [inline cypress/README.md]:
```suggestion
Unit tests are written and maintained by developers.
```

File: cypress/README.md [hunk 3]
```diff
@@ -1,4 +1,30 @@
...
+  * Internet Explorer 11
+4. Jira:
+  * For every testable task needs to be created test sessions per every browser
+  * Results should be added to the created test sessions with pass / fail mark
+  * If the result is set to fail the issue should be raised in Jira
+
+# Cypress Cucumber tests for Carbon storybook (Automation Testing)
+
+## Description
+  * Automation testing are writing using Cypress.io framework.
```
Comment (jamime) [inline cypress/README.md]:
```suggestion
  * Automation tests are written using [Cypress.io](https://www.cypress.io/) framework.
```

File: cypress/README.md (hunk 1 above)
Comment (jamime) [inline cypress/README.md]:
```suggestion
2. Each component should have a dedicated story (default / classic / multiple / validation / themes etc.), notes section and accessibility section.
```

File: cypress/README.md [hunk 4]
```diff
@@ -1,4 +1,30 @@
...
+1. Every component should be fully tested using functional testing / UI testing / Internationalisation testing
+2. Should be tested all stories for dedicated component (default / classic / multiple / validation / themes etc.), notes section and accessibility section.
+3. Supporting browsers:
+  * Chrome
+  * Firefox
+  * Safari
+  * Edge
+  * Internet Explorer 11
+4. Jira:
+  * For every testable task needs to be created test sessions per every browser
```
Comment (jamime) [inline cypress/README.md]:
```suggestion
  * For every testable task, a JIRA test session needs to be created for each of the supported browsers.
```

File: cypress/README.md [hunk 5]
```diff
@@ -1,4 +1,30 @@
...
+  * Results should be added to the created test sessions with pass / fail mark
+  * If the result is set to fail the issue should be raised in Jira
```
Comment (jamime) [inline cypress/README.md]:
```suggestion
  * Any issues found during testing should be raised as a "Defect" JIRA.
  * If the issue is present on the `master` branch it should be raised as a "Bug" in JIRA, not "Defect".
```

File: cypress/README.md (hunk 1 above)
Comment (DlgSHi) [reply cypress/README.md]: fixed

File: cypress/README.md (hunk 2 above)
Comment (DlgSHi) [reply cypress/README.md]: updated

File: cypress/README.md (hunk 1 above)
Comment (DlgSHi) [reply cypress/README.md]: updated

File: cypress/README.md (hunk 5 above)
Comment (DlgSHi) [reply cypress/README.md]: updated

File: cypress/README.md (hunk 4 above)
Comment (DlgSHi) [reply cypress/README.md]: updated

File: cypress/README.md [hunk 6]
```diff
@@ -1,51 +1,79 @@
-# Cypress Cucumber tests for Carbon storybook
+# Testing Strategy
+
+## Unit Testing
+   Unit tests are written and maintained by developers.
+
+## Manual Testing
+### Components:
+1. Every component should be fully tested using functional testing / UI testing / Internationalisation testing.
+2. Each component should have a dedicated story (default / classic / multiple / validation / themes etc.),
```
Comment (jamime) [inline cypress/README.md]:
```suggestion
2. Every component should have a dedicated story (default / classic / multiple / validation / themes etc.),
```

File: cypress/README.md (hunk 3 above)
Comment (DlgSHi) [reply cypress/README.md]: updated

File: cypress/README.md (hunk 6 above)
Comment (DlgSHi) [reply cypress/README.md]: fixed

File: cypress/README.md [hunk 7]
```diff
@@ -1,51 +1,79 @@
...
+4. Jira:
+  * For every testable task, a JIRA test session needs to be created for each of the supported browsers;
+  * Results should be added to the created test sessions with pass / fail mark;
+  * Any issues found during testing should be raised as a "Defect" JIRA;
+  * If the issue is present on the `master` branch it should be raised as a "Bug" in JIRA, not as a "Defect".
+
+# Cypress Cucumber tests for Carbon storybook (Automation Testing)
+
+## Description
+  * Automation tests are written using [Cypress.io] (https://www.cypress.io/) framework.
```
Comment (jamime) [inline cypress/README.md]:
```suggestion
  * Automation tests are written using [Cypress.io](https://www.cypress.io/) framework.
```

Comment (DlgSHi) [reply cypress/README.md]: updated

**Pull request merged**